packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -89,26 +93,18 @@ class SimpleFluidError implements IFluidErrorBase {
         this.message = errorProps.message;
         this.stack = errorProps.stack;
         this.name = errorProps.name;
+
+        //* PR: Do I need to avoid adding optional ones, or is it ok to add an explicit undefined value?
+        // (like I wonder if it will show up in Kusto as "undefined" -- Oh I think not, recalling the Bohemia code)
+        this.addTelemetryProperties(errorProps);
     }
 
     getTelemetryProperties(): ITelemetryProperties {
-        const props: ITelemetryProperties = {
-            ...this.telemetryProps,
-            errorType: this.errorType,
-            fluidErrorCode: this.fluidErrorCode,
-            message: this.message,
-        };
-        if (this.name !== undefined) {
-            props.name = this.name;
-        }
-        if (this.stack !== undefined) {
-            props.stack = this.stack;
-        }
-        return props;
+        return this.telemetryProps;
     }
 
     addTelemetryProperties(props: ITelemetryProperties) {
-        copyProps(this.telemetryProps, props);
+        copyProps(props, this.telemetryProps);
```

Comment:
Nit: I think the convention I've seen is that the 'target' is the first arg.

Reply:
can we name this more explicitly, even source and target are not super clear to me